### ISSUE TITLE
Error when importing from namespace package using `from`-style imports

### ISSUE THREAD

**Issue by russelldavis** [microsoft/pyright#850]
To reproduce:
```sh
mkdir repro
cd repro
echo "from lib import bar" > foo.py
touch lib/bar.py
pyright foo.py
```

You'll get this error: `error: Import "lib" could not be resolved (reportMissingImports)`.
But the import works fine.

Note that if you `touch lib/__init__.py`, the pyright error goes away. It also goes away if you change the import to `import lib.bar`. So it's a problem specifically with namespace packages and `from`-style imports.

**Comment by erictraut** [comment]
Thanks for the bug report. 

You are correct — there was a bug in the import resolution logic relating to local namespace packages. It will be fixed in the next release.

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.54, which I just published. It will also be included in the next release of Pylance.
